card/lib/card/migration.rb

```diff
@@ -70,6 +70,10 @@ def contentedly
       end
     end
 
+    def disable_ddl_transaction #:nodoc:
+      true
```

Comment:
probably, but that's still better than migrations that don't work at all. I think I did that to fix the  postgres migration issues but I couldn't test it because I didn't manage to start a postgres deck. Did you find a solution?